snippets/country-localization.liquid

```diff
@@ -13,10 +13,12 @@
     aria-controls="{{ localPosition }}List"
     aria-describedby="{{ localPosition }}Label"
   >
-    <span>
-      {{- localization.country.currency.iso_code }}
-      {{ localization.country.currency.symbol }} | {{ localization.country.name -}}
-    </span>
+    
+    <span  class="localization-form__country_flag">
+      {{ localization.country | image_url: width: 18 | image_tag }} 
+      <span style="padding-left: 1rem"> {{- localization.country.currency.iso_code }} <span>
```

Comment:
I wouldn't go with inline styling here. We want to default to CSS files as much as possible otherwise it's more things to keep track of and it impacts CSS specificity.